### ISSUE TITLE
Per-cluster ingress creation should be idempotent

### ISSUE THREAD

**Issue by madhusudancs** [GoogleCloudPlatform/k8s-multicluster-ingress#18]
I created ingresses in my clusters as part of service creation. I had all my manifest files in a directory and I ran `kubectl create -f manifests/`. Now when I run `kubemci` as:

```shell
./kubemci create hello --ingress=manifests/ingress.yaml --gcp-project=myproject --kubeconfig=mcikubeconfig 
```

I get the following error:

```
Error in creating load balancer: 3 errors occurred:

* Error in creating ingress in cluster cluster1: ingresses.extensions "nginx" already exists
* Error in creating ingress in cluster cluster2: ingresses.extensions "nginx" already exists
* Error in creating ingress in cluster cluster3: ingresses.extensions "nginx" already exists
```

`kubemci create` is supposed to be idempotent and this error is unexpected.

**Comment by madhusudancs** [comment]
cc @G-Harmon

**Comment by G-Harmon** [comment]
I think this should be fixed now, by PR #32

**Comment by nikhiljindal** [comment]
yes verified that this is fixed now
